### PR TITLE
Create senyang.md

### PR DESCRIPTION
**Pull request by wsy-239**

### CODE REVIEW

File: week1-introduction/homework/wsy.md [hunk 1]
```diff
@@ -0,0 +1 @@
+test
```
Comment (stargrep) [inline week1-introduction/homework/wsy.md:1]: you should change this file name to {first_name}.md instead of 'wsy.md'. I will approve after.

Comment (wsy-239) [reply week1-introduction/homework/wsy.md]: I have changed the file name.

Comment (CodeSEAS) [reply week1-introduction/homework/wsy.md]: Great, LGTM!

Comment (stargrep) [reply week1-introduction/homework/wsy.md]: Great, LGTM

**Pull request merged**